venues/thecvf.com/ECCV/2020/Conference/python/post-custom-load-edges.py

```diff
@@ -1,130 +1,129 @@
 
-confirmations = {}
-confirmation_notes = openreview.tools.iterget_notes(
-            client, 
-            invitation='thecvf.com/ECCV/2020/Conference/{grp}/-/Profile_Confirmation'.format(grp=reviewer_group_name))
-for note in tqdm(confirmation_notes):
-    # Check if this user has posted multiple confirmations
-    if note.tauthor in confirmations:
-        # Check if note is a newer confirmation
-        if note.tcdate > confirmations[note.tauthor].tcdate:
-            confirmations[note.tauthor] = note
-    else:
-        confirmations[note.tauthor] = note
-
-print ('Confirmations received: ', len(confirmations))
-
-custom_loads = {}
-custom_load_notes = openreview.tools.iterget_notes(
-    client, 
-    invitation='thecvf.com/ECCV/2020/Conference/-/Reduced_Load')
-
-for note in tqdm(custom_load_notes):
-    # Check if this user has posted multiple reduced load notes
-    if note.content['user'] in custom_loads:
-        # Check if note is a newer confirmation
-        if note.tcdate > custom_loads[note.content['user']].tcdate:
-            custom_loads[note.content['user']] = note
-    else:
-        custom_loads[note.tauthor] = note
-
-print ('Reduced loads received: ', len(custom_loads))
-
-
-profile_map = {}
-
-emails = []
-tildes = []
-for ac in reviewer_group.members:
-    if ac.startswith('~'):
-        tildes.append(ac)
-    else:
-        emails.append(ac)
-
-active = 0
-tilde_profiles=client.search_profiles(ids=tildes)
-email_profiles=client.search_profiles(emails=emails)
-
-inactives = []
-
-for profile in tilde_profiles:
-    profile_map[profile.id] = profile
-    if profile.active and profile.password:
-        active+=1
-    else:
-        inactives.append(profile.id)
-
-for member in email_profiles:
-    profile_map[member] = email_profiles[member]
-    if profile_map[member].active and profile_map[member].password:
-        active+=1
-    else:
-        inactives.append(member)
-
-
-print ('Number of active profiles:', active)
-print ('Number of inactive profiles:', len(inactives))
-
-miss = 0
-total_review_capacity = 0
-
-custom_load_edges = []
-
-for reviewer in tqdm(reviewer_group.members):
-    review_capacity = 0
-    
-    profile = profile_map.get(reviewer, None)
-    if not profile:
-        miss += 1
-        profile = openreview.tools.get_profile(client, reviewer)
-
-    confirmation = None
-    custom_load = None
-    if profile:
+    for note in tqdm(custom_load_notes):
+        # Check if this user has posted multiple reduced load notes
+        if note.content['user'] in reduced_loads:
+            # Check if note is a newer confirmation
+            if note.tcdate > reduced_loads[note.content['user']].tcdate:
+                reduced_loads[note.content['user']] = note
+        else:
+            reduced_loads[note.content['user']] = note
+
+    print ('Reduced loads received: ', len(reduced_loads))
+
+    profile_map = {}
+
+    emails = []
+    tildes = []
+    for ac in reviewer_group.members:
```

Comment:
we should work only with profiles, emails are reviewers with no profile and they will be removed before the paper assignment

Reply:
Fixed.